### PR TITLE
Implement gRPC service router.

### PR DESCRIPTION
**Pull request by muzykantov**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,7 +22,6 @@
 
 # Dependency directories (remove the comment below to include it)
 vendor/
-Godeps/
```
Comment (pfi79) [inline .gitignore:25]: why did you delete godeps?

Comment (muzykantov) [reply .gitignore:25]: had to restart the pipline. found something unused.

**Review by pfi79 — APPROVED** [review]
